examples/sn-blog-gatsby/src/pages/index.tsx

```diff
@@ -58,7 +59,6 @@ export interface IndexPageProps {
 const IndexPage: React.FC<IndexPageProps> = (props) => {
   const { data } = props
   const classes = useStyles()
-
   return (
     <IndexLayout>
       <Page>
```

Comment:
Probably this component is replacable with a class on the container.
If you remove this and LayoutRoot also, container could be rendered as main html tag on 404 and pages/index, article tag on templates/page